main.py

```diff
@@ -800,17 +801,18 @@ def handle_interface_link_down(self, interface):
         """
         Handler for interface link_down events
         """
+        log.info("Event handle_interface_link_down %s", interface)
         for evc in self.get_evcs_by_svc_level():
             with evc.lock:
-                log.info("Event handle_interface_link_down %s", interface)
```

Comment:
took the opportunity to fix the placement of this logging message, it was annoying to see so many messages on the log :)